src/EnergyPlus/DataEnvironment.cc

```diff
@@ -298,7 +254,7 @@ void SetOutBulbTempAt_error(EnergyPlusData &state, std::string const &Settings,
     ShowFatalError(state, "Program terminates due to preceding condition(s).");
 }
 
-void SetWindSpeedAt(EnergyPlusData &state,
+void SetWindSpeedAt(EnergyPlusData const &state,
                     int const NumItems,
                     const Array1D<Real64> &Heights,
                     Array1D<Real64> &LocalWindSpeed,
```

Comment:
[src/EnergyPlus/DataEnvironment.cc:301]:(style),[constParameter],Parameter 'state' can be declared as reference to const

Reply:
:+1: